style.css

```diff
@@ -419,14 +419,177 @@ input:focus {
   border-color: rgb(98, 108, 201);
 }
 
+.headerlish {
+  display: none;
+}
+
 @media (min-width: 375px) {
   .content-section {
     background-size: cover;
   }
 }
 
+/* desktop version */
+
 @media (min-width: 768px) {
+  .icon-Union {
+    display: none;
+  }
+
+  .topBar-nav {
+    position: sticky;
+    top: 0;
+    background-color: white;
+  }
+
+  .headerlist {
+    display: flex;
+    gap: 8px;
+    list-style-type: none;
+    padding-right: 20px;
+    font-family: var(--primary-font);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    position: sticky;
+    top: 0;
+  }
+
+  .headerlist a:hover {
+    color: rgb(24, 155, 160);
+  }
+
   .content-section {
+    background-image: url(img/Header\ bg-desktop.png);
+    background-size: contain;
+    display: flex;
+    height: 718px;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .main-Container {
+    width: 60%;
+    margin-left: 0;
+    padding-top: 0;
+  }
+
+  .work-container {
+    display: flex;
+    margin: 30px 10px;
+  }
+
+  .cards {
+    width: auto;
+    flex-direction: row;
+    gap: 25px;
+    align-items: center;
+    justify-content: center;
+  }
```

Comment:
- [ ] You've made some changes to the design of the cards for the desktop version of your project, which is great :+1:. However, to ensure that it matches the Figma template, there are a few things you should consider:
- Please make sure that the background color of the works section is #E5E5E5, which is the same as shown in the template.
- The cards should be centered on the page and the width and height of each card should be adjusted accordingly.
- It would be a good idea to check the spacing between each card to ensure consistency throughout the page.

**Your design**

![Screenshot from 2023-04-01 16-07-55](https://user-images.githubusercontent.com/105117832/229314087-95e66ce9-1a02-4f3c-b9ff-0f23ec7083ef.png)

**Figma**

![Screenshot from 2023-04-01 16-09-00](https://user-images.githubusercontent.com/105117832/229314094-710db297-16f3-44b6-97ca-bcbe2d097b63.png)